src/phoenix/core/evals.py

```diff
@@ -102,3 +116,21 @@ def get_document_evaluations_by_span_id(self, span_id: SpanID) -> List[pb.Evalua
             for evaluations in self._document_evaluations_by_span_id[span_id].values():
                 all_evaluations.extend(evaluations.values())
         return all_evaluations
+
+    def get_document_evaluation_scores(
+        self,
+        span_id: SpanID,
+        evaluation_name: str,
+        num_documents: int,
+    ) -> List[float]:
+        # num_documents is needed as argument because the document position values
+        # are not checked during ingestion: e.g. if there exists a position value
+        # of one trillion, we would not want to create a result that large.
```

Comment:
Good question. This is done mainly to de-couple Traces from Evals. I think this minor inconvenience is worthwhile because it let's proceed with our development in the most light-weight fashion. On the GraphQL side, we can also make it a private variable if we want, that way it won't actually pollute the API. So there is also that.